src/core/components/parameters/requestUrlOutSystems.jsx

```diff
@@ -0,0 +1,34 @@
+//OutSystems change - component to render the request URL information
+import React from "react"
```

Comment:
There are a lot of missing `;` at the end of the statements in this file 😬

Reply:
done